### ISSUE TITLE
Composer laravel/framework updates not happening

### ISSUE THREAD

**Issue by Jamesking56** [renovatebot/renovate#4777]
**What Renovate type are you using?**

Docker image run via GitLab CI.

**Describe the bug**

`laravel/framework` has updates available that are not being applied by Renovate. Other updates seem to be working as normal.

**Did you see anything helpful in debug logs?**

It just shows the dependency as having no updates. My `renovate.json` config is:

```json
{
    "extends": [
        "config:base",
        ":assignAndReview(Jamesking56)",
        ":rebaseStalePrs",
        ":preserveSemverRanges"
    ],
    "dockerfile": {
        "enabled": false
    },
    "gitlabci": {
        "enabled": false
    }
}
```

`laravel/framework` is set to `^6.2` in `composer.json`. Laravel `6.2.0` is currently in `composer.lock` and `6.2.1` is available but being ignored by Renovate (for some strange reason).

**To Reproduce**

Basically copy the config from above.

**Additional context**

**Comment by rarkins** [comment]
That sounds expected, because 6.2.1 is satisfied by `^6.2`. If you want PRs for every update then you need to pin your versions in `composer.json`. Renovate will only send PRs for Composer when the composer.json needs updating. Lockfile-only PRs are not supported.

**Comment by Jamesking56** [comment]
> Lockfile-only PRs are not supported.

Are there any plans to support this? Dependabot for example does this

**Comment by rarkins** [comment]
We support it for npm and yarn so far, but not composer. Why not just pin your dependency version in composer.json?

**Comment by Jamesking56** [comment]
I suppose I could pin the dependencies and remove `composer.lock` entirely. How come lockfile updates are available for npm and yarn? Can't you do dependency pinning with those as well and remove lockfiles?

**Comment by rarkins** [comment]
I don't suggest that you remove the composer.lock. But you usually don't gain anything by including ranges in composer.json unless you're publishing a library.

**Comment by rarkins** [comment]
You pin dependencies in your package file so that it's clear to you and other developers about what direct versions you're using. The lock file is still useful to lock/freeze all the indirect dependencies.
